index.html

```diff
@@ -60,6 +60,57 @@ <h1>Hello, world!</h1>
         <p><a class="btn btn-primary btn-lg" href="#" role="button">Learn more &raquo;</a></p>
       </div>
     </div>
+    
+    <div class="container">
+      <div class="row">
+        <div class="col-md-4">
+          <h2>Search for Poetry by Genre</h2>
+          <form role="search" action="submit">
+              <label for="search-field">
```

Comment:
I think having a search field is a great idea! The search by genre form will allow the project to be easily narrowed down to an area or specific poem a user might be looking to filter through. Good use of using label and the form HTML tags for this!